lib/utils/setDeclarationValue.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const _ = require('lodash');
+
+/** @typedef {import('postcss').Declaration} Declaration */
+
+/**
+ * @param {Declaration} decl
+ * @param {string} value
+ * @returns {Declaration} The declaration that was passed in.
+ */
+module.exports = function getDeclarationValue(decl, value) {
```

Comment:
Oh, this is a typo. 😅 

```suggestion
module.exports = function setDeclarationValue(decl, value) {
```